projects/plugins/jetpack/package.json

```diff
@@ -180,14 +180,17 @@
 		"eslint": "7.31.0",
 		"fs-extra": "10.0.0",
 		"glob": "7.1.6",
+		"install": "0.13.0",
```

Comment:
What's this for?

Reply:
That's weird, must have been put in there by `pnpm up ...` at some point. I didn't add it. I'll see if things break when I remove that and the other spurious dependencies.